tests/Feature/ApiRateLimitTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Tests\TestCase;
+
+class ApiRateLimitTest extends TestCase
+{
+
+    public function testRateLimit()
+    {
+        config(['app.api_throttle_per_minute' => 10]);
+        $this->actingAsForApi(User::factory()->create())
+            ->getJson(route('api.users.me'))
+            ->assertOk()
+            ->assertHeader('X-Ratelimit-Limit', config('app.api_throttle_per_minute'))
+            ->assertHeader('X-Ratelimit-Remaining', 9);
+    }
+
+    public function testRateLimitDecreasesRemaining()
+    {
+        config(['app.api_throttle_per_minute' => 5]);
+        $expected_remaining = (config('app.api_throttle_per_minute') - 1);
+        $admin = User::factory()->create();
+
+        for ($x = 0; $x < 5; $x++) {
+
+            $this->actingAsForApi($admin)
+                ->getJson(route('api.users.me'))
+                ->assertOk()
+                ->assertHeader('X-Ratelimit-Remaining', $expected_remaining--);
+
+        }
+
+        $this->actingAsForApi($admin)
+            ->getJson(route('api.users.me'))
+            ->assertStatus(429)
+            ->assertHeader('Retry-After', 60);
```

Comment:
Just for testing completeness, can we do a similar `config()` call for some weird value for `Retry-After` similarly, above, and make sure that shows back up here? Just to add another test point here.

Reply:
That rate limit is tied directly into the Route Service Provider which locks it in at 60 seconds, so I don't think we need that test.